widget/playground/src/containers/DefaultChainAndToken/DefaultChainAndToken.tsx

```diff
@@ -0,0 +1,124 @@
+import type { Type } from '../../types';
+
+import { ChainsIcon, Divider } from '@rango-dev/ui';
+import React, { useState } from 'react';
+
+import { ItemPicker } from '../../components/ItemPicker';
+import { OverlayPanel } from '../../components/OverlayPanel';
+import { SingleList } from '../../components/SingleList';
+import { tokensAreEqual, tokenToString } from '../../helpers';
+import { useConfigStore } from '../../store/config';
+import { useMetaStore } from '../../store/meta';
+import { ModalState } from '../FunctionalLayout/FunctionalLayout.types';
+
+export function DefaultChainAndToken({ type }: { type: Type }) {
+  const [modalState, setModalState] = useState<ModalState | null>(null);
+  const {
+    config: { from, to },
+    onChangeBlockChain,
+    onChangeToken,
+  } = useConfigStore();
+  const {
+    meta: { blockchains, tokens },
+  } = useMetaStore();
+
+  const selectedType = type === 'Source' ? from : to;
+
+  const filteredTokens = selectedType?.blockchain
+    ? tokens.filter((token) => token.blockchain === selectedType.blockchain)
+    : [];
+  const chainValue = blockchains.find(
+    (chain) => chain.name === selectedType?.blockchain
+  );
+  const tokenValue = tokens.find((token) =>
+    tokensAreEqual(token, selectedType?.token)
+  );
+
+  const handleDefaultBlockchainConfirm = (item: string) => {
+    onChangeBlockChain(item, type);
+    onChangeToken(undefined, type);
+
+    onBack();
+  };
+
+  const handleDefaultTokenConfirm = (item: string) => {
+    const selectedToken = filteredTokens.find(
+      (token) => tokenToString(token) === item
+    );
+    if (selectedToken) {
+      onChangeToken(
+        {
+          blockchain: selectedToken.blockchain,
+          address: selectedToken.address,
+          symbol: selectedToken.symbol,
+        },
+        type
+      );
+    }
+
+    onBack();
+  };
+
+  const onBack = () => setModalState(null);
+
+  const filteredBlockchains = selectedType?.blockchains
+    ? blockchains.filter((chain) =>
+        selectedType.blockchains?.includes(chain.name)
+      )
+    : blockchains;
+
+  return (
+    <>
+      <ItemPicker
```

Comment:
While testing on chrome zoom:+175%, scroll behavior is not working as expected.

![Recording 2023-10-21 at 12 16 24](https://github.com/rango-exchange/rango-client/assets/99180934/9ccba3e6-3a44-4af4-a434-59d1429c8968)

Reply:
fixed.